.github/workflows/extended-ci-longevity-large-partition-asymmetric-cluster-3h-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Build scylla-bench docker image with gocql PR
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled]
+
+jobs:
+  trigger-longevity-large-partition-asymmetric-cluster-3h-test:
+    if: contains(github.event.pull_request.labels.*.name, 'extended-ci')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Check out the scylla-bench repository
+        uses: actions/checkout@v2
+        with:
+          repository: scylladb/scylla-bench
+          path: scylla-bench
+
+      - name: Checkout GoCQL PR Repository
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          path: gocql
+
+      - name: Build and push Scylla-bench Docker Image with gocql from PR
+        run: |
+          cd scylla-bench
+          GOCQL_REPO="github.com/${{ github.event.pull_request.head.repo.full_name }}" GOCQL_VERSION="${{ github.event.pull_request.head.ref }}" make build-with-custom-gocql-version
+          DOCKER_IMAGE_TAG="scylladb/gocql-extended-ci:scylla-bench-${{ github.event.pull_request.head.sha }}" DOCKER_IMAGE_LABELS="com.scylladb.gocql-version=${{ github.event.pull_request.head.sha }}" make build-sct-docker-image
+          docker push "scylladb/gocql-extended-ci:scylla-bench-${{ github.event.pull_request.head.sha }}"
+
+      - name: Start Jenkins job
+        uses: sylwiaszunejko/jenkins_client@main
+        with:
+          jenkins_job_name: 'scylla-drivers/gocql/extended-ci/longevity-large-partition-asymmetric-cluster-3h-test'
+          jenkins_job_parameters: '{"email_recipients": "sylwia.szunejko@scylladb.com", "scylla_version": "6.1.1", "extra_environment_variables": "SCT_STRESS_IMAGE.scylla-bench=scylladb/gocql-extended-ci:scylla-bench-${{ github.event.pull_request.head.sha }}"}'
```

Comment:
> I'm unsure if we should fix the scylla_version to something specific. On one hand, it can make the CI more predictable (not sure if stable, but quite repeatable). On the other hand, at some point we need to go back and update it and possibly face new issues.

having `master` there is not an option.
We can do same thing we do for `cpp-rust-driver` - get last release and run on it.

> BTW, there is an option to set here "scylla_version: 6.1" so it will take all scylla 6.1 patches. Another option is to use an enterprise release like 2024.1 who is LTS and won't require us to update it very often (will automatically pick the latest patch release).

From one side using OSS will bring top feature to the test as fast as possible.
On other side using enterprise LTS will bring more stability.
It looks like we need both ))

> Regarding email_recipent, can we set it to the author of the PR (as long as it internal author?) - not a must.

It is doable, but it will expose ip addresses, urls of our test infrastructure, if we are ok with it then sure, why not.
But we also should have scylla-drivers@scylladb.com as CC.

> Lastly, I noticed that the Jenkins job is set to master, which again brings back the question of stability vs being updated automatically.

Agree, it is better to target either `branch-6.1` or `branch-2023` or particular commit from master

Reply:
> > I'm unsure if we should fix the scylla_version to something specific. On one hand, it can make the CI more predictable (not sure if stable, but quite repeatable). On the other hand, at some point we need to go back and update it and possibly face new issues.
> 
> having `master` there is not an option.
> We can do same thing we do for `cpp-rust-driver` - get last release and run on it.
> 
> > BTW, there is an option to set here "scylla_version: 6.1" so it will take all scylla 6.1 patches. Another option is to use an enterprise release like 2024.1 who is LTS and won't require us to update it very often (will automatically pick the latest patch release).
> 
> From one side using OSS will bring top feature to the test as fast as possible.
> On other side using enterprise LTS will bring more stability.
> It looks like we need both ))
> 
> > Regarding email_recipent, can we set it to the author of the PR (as long as it internal author?) - not a must.
> 
> It is doable, but it will expose ip addresses, urls of our test infrastructure, if we are ok with it then sure, why not.
> But we also should have scylla-drivers@scylladb.com as CC.

Nothing secret in the emails
Also those should run only by trusted users (with some definition of trust), isn't it ?

> 
> > Lastly, I noticed that the Jenkins job is set to master, which again brings back the question of stability vs being updated automatically.
> 
> Agree, it is better to target either `branch-6.1` or `branch-2023` or particular commit from master
> 

release branch do get backports, on the other hand they might not have all of the fixes features for those test (not to mention we'll need to backport specific things unrelated to a release need)

A specific master commit, is also problematic, when one will update it ? what happens when it breaks and stops working ?